### PR TITLE
feat(automl): generate v1

### PR DESCRIPTION
**Pull request by busunkim96**

### CODE REVIEW

File: automl/docs/index.rst [hunk 1]
```diff
@@ -1,8 +1,16 @@
...
+v1 API Reference
+------------------------
+.. toctree::
+    :maxdepth: 2
+
+    gapic/v1/api
+    gapic/v1/types
+
+v1beta1 API Reference
+----------------------
```
Comment (tseaver) [inline automl/docs/index.rst]: Does this need language like other "old API version" blocks?  See, for instance, the [scheduler docs](https://github.com/googleapis/google-cloud-python/blob/master/scheduler/docs/index.rst).

**Comment by tseaver** [comment]
@busunkim96 How are you fixing the b0rken changes to `setup.py`?  When I run `synthtool` inside the `automl/` directory, I'm still seeing that change.

**Comment by busunkim96** [comment]
Is something off with the setup.py? I think the synth script is written to not copy it 

https://github.com/googleapis/google-cloud-python/blob/f068eb8be6e8f86cdec4d6e375307f1b425c1caf/automl/synth.py#L27-L34

**Comment by tseaver** [comment]
@busunkim96 I mistyped:  the bad changes I'm seeing are in `docs/conf.py`.

**Comment by busunkim96** [comment]
Ah, I just manually avoided committing it. :) Hopefully the monolith is released soon.

**Comment by tseaver** [comment]
Looks like the fix for the requests intersphinx URL finally landed in artman today:  maybe you were using a pre-released version?

**Pull request merged**